### ISSUE TITLE
ReqlDriverError: `r.or` takes at least 2 arguments is not compatible with official API. 

### ISSUE THREAD

**Issue by gutenye** [neumino/rethinkdbdash#157]
According to the official API http://rethinkdb.com/api/javascript/or/, `r.or(bool[, bool, ...]) ` works with one argument.

And in some case, one argument is needed. For example

```javascript
var list = ["a"] or ["a", "b"]
list = list.map(v => r.row("name").eq(v))
r.expr([{name: "b"}]).filter(r.or(...list))
```

Given `list = ["a", "b"]`, it works fine.
Given `list = ["a"]`, it raises this error.

**Comment by neumino** [comment]
Fixed in 2.1.9 (with `r.and` too)

**Comment by neumino** [comment]
Thanks @gutenye for the report :)
